src/errors.rs

```diff
@@ -1,26 +1,62 @@
+/// Alias for [`core::result::Result`] with the `rsa` crate's [`Error`] type.
 pub type Result<T> = core::result::Result<T, Error>;
 
 /// Error types
 #[derive(Debug, Eq, PartialEq)]
 #[non_exhaustive]
 pub enum Error {
+    /// Invalid padding scheme.
```

Comment:
Alternatively we could disable the lint for this type, if you think the variant names are self-explanatory.

Reply:
Nah, let's not disable the linter.